apps/openmw/mwworld/inventorystore.cpp

```diff
@@ -272,7 +272,7 @@ void MWWorld::InventoryStore::autoEquip (const MWWorld::Ptr& actor)
         }
 
         if (iter.getType() == ContainerStore::Type_Armor &&
-                test.getClass().getEffectiveArmorRating(test, actor) <= std::max(unarmoredRating, 0.f))
+                test.getClass().getEffectiveArmorRating(test, actor) < std::max(unarmoredRating, 0.f))
```

Comment:
The problem is elsewhere. See https://github.com/OpenMW/openmw/pull/1215 for details.